js/renovation/component_wrapper/component.ts

```diff
@@ -158,23 +151,53 @@ export default class ComponentWrapper extends DOMComponent {
     return this._elementAttr;
   }
 
+  _setDefaultOptionValue(options, defaultValueGetter) {
```

Comment:
This method does not use this, so why is it an instance method now?

Reply:
just forgot to move it out after some changes, thx